<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddressConverterView</class>
 <widget class="QWidget" name="AddressConverterView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>781</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QSplitter" name="addr_con_splitter">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string/>
      </property>
      <widget class="QLineEdit" name="lineEdit">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>133</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(85, 170, 0);</string>
       </property>
       <property name="text">
        <string>RVA</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>133</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 170, 0);</string>
       </property>
       <property name="text">
        <string>VA</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_va">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>0</y>
         <width>133</width>
         <height>21</height>
        </rect>
       </property>
       <property name="clearButtonEnabled">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_rva">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>20</y>
         <width>133</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_5">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>40</y>
         <width>133</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 0);</string>
       </property>
       <property name="text">
        <string>File Offset</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_file_offset">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>40</y>
         <width>133</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="HexEditorView" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>761</width>
         <height>69</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HexEditorView</class>
   <extends>QWidget</extends>
   <header>view/hexeditorview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
